package.json

```diff
@@ -34,16 +34,21 @@
     "@fortawesome/react-fontawesome": "^0.1.12",
     "@hookform/resolvers": "^2.9.7",
     "@sentry/nextjs": "^6.5.0",
+    "@types/react-draft-wysiwyg": "^1.13.4",
     "@xstate/react": "^1.3.1",
     "base-64": "^1.0.0",
     "bootstrap": "^4.5.3",
     "clipboard-copy": "^4.0.1",
     "crypto-browserify": "^3.12.0",
     "date-fns": "^2.16.1",
     "deep-object-diff": "^1.1.0",
+    "draft-js": "^0.11.7",
+    "draftjs-to-html": "^0.9.1",
     "fastest-levenshtein": "^1.0.12",
+    "html-to-draftjs": "^1.5.0",
     "i18next": "^19.8.3",
     "i18next-browser-languagedetector": "^6.0.1",
+    "immutable": "^4.2.4",
```

Comment:
Didn't it work without specifying this specific version? 
Since it is included when defining the `draft-js` dependency

Reply:
👍🏻